cortx-monitorQuickstartGuide.md

```diff
@@ -49,8 +49,8 @@ This guide provides a step-by-step walkthrough for getting you CORTX-Monitor Ser
 
         :page_with_curl: **Notes:** 
         
-      - If the https://raw.githubusercontent.com/Seagate/cortx-prvsnr/dev/cli/src/cortx-prereqs.sh?token=APA75GY34Y2F5DJSOKDCZAK7ITSZC link is not accessible, generate new one.
-      - Visit https://github.com/Seagate/cortx-prvsnr/blob/dev/cli/src/cortx-prereqs.sh and naviagte to RAW > Copy URL > Use the URL for deployment.
+      - If the https://raw.githubusercontent.com/Seagate/cortx-re/main/scripts/third-party-rpm/install-cortx-prereq.sh?token=APBA7EBWCTPTEOH5PPWFYYTBCN23Y link is not accessible, generate new one.
+      - Visit https://github.com/Seagate/cortx-re/blob/main/scripts/third-party-rpm/install-cortx-prereq.sh and naviagte to RAW > Copy URL > Use the URL for deployment.
```

Comment:
Please correct following typos:
Line 53: naviagte -> navigate
Line 78: 1. Before you build the RPM, you'll need to install the `dependencies` using:

Reply:
Done.